nodes/src/nodes/multi_agent/orchestrator.py

```
@@ -0,0 +1,549 @@
+                            task_id=task.id,
+                            error=f'Unknown agent: {task.assigned_agent}',
+                        )
+                        continue
+                    future = pool.submit(self._execute_agent, agent, task.description, task.id)
+                    futures[future] = task
+
+                for future in as_completed(futures):
+                    task = futures[future]
+                    try:
+                        result = future.result()
+                    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)